### PR TITLE
HOTFIX: Fix broker bounce system tests

### PR DESCRIPTION
**Pull request by cadonna**
### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Comment by cadonna** [comment]
Call for review: @mjsax @abbccdda

File: tests/kafkatest/tests/streams/streams_broker_bounce_test.py [hunk 1]
```diff
@@ -164,7 +164,7 @@ def setup_system(self, start_processor=True, num_threads=3):
 
         # Start test harness
         self.driver = StreamsSmokeTestDriverService(self.test_context, self.kafka)
-        self.processor1 = StreamsSmokeTestJobRunnerService(self.test_context, self.kafka, num_threads)
+        self.processor1 = StreamsSmokeTestJobRunnerService(self.test_context, self.kafka, "at_least_once", num_threads)
```
Comment (cadonna) [inline tests/kafkatest/tests/streams/streams_broker_bounce_test.py:167]: Is it enough to specify the processing guarantee as `at_least_once` here or do you want also for this test to include all processing guarantees in the test matrix?

Comment (cadonna) [reply tests/kafkatest/tests/streams/streams_broker_bounce_test.py:167]: On a different note, now that the processing guarantee can be passed to the service, do we still need `StreamsEosTestJobRunnerService` and `StreamsComplexEosTestJobRunnerService`?

**Comment by cadonna** [comment]
Streams system tests run: https://jenkins.confluent.io/job/system-test-kafka-branch-builder/3914

**Comment by cadonna** [comment]
Streams system tests above are green.

**Review by abbccdda — APPROVED** [review]
LGTM!

File: tests/kafkatest/tests/streams/streams_broker_bounce_test.py (hunk 1 above)
Comment (abbccdda) [reply tests/kafkatest/tests/streams/streams_broker_bounce_test.py:167]: I feel neutral about it, if we are adding 2X time to system tests, that's probably not good.

Comment (mjsax) [reply tests/kafkatest/tests/streams/streams_broker_bounce_test.py:167]: We can try to add both eos cases, too.

For `Streams[Complex]EosTestJobRunnerService`, they use a different Java class compared to the "smoke test" runner. Not sure if we could unify them?

**Pull request merged**